src/renderer/components/ft-playlist-add-video-prompt/ft-playlist-add-video-prompt.js

```diff
@@ -0,0 +1,258 @@
+        if (playlist.createdAt == null) { return }
+        // Only playlists created after this prompt shown should be considered
+        if (playlist.createdAt < this.lastShownAt) { return }
+        // Only playlists not auto added to selected yet should be considered
+        if (this.createdSincePromptShownPlaylistIdList.includes(playlist._id)) { return }
+
+        // Add newly created playlists to selected ONCE
+        this.createdSincePromptShownPlaylistIdList.push(playlist._id)
+        this.selectedPlaylistIdList.push(playlist._id)
+      })
+
+      // Remove  deleted playlist from deleted
+      this.selectedPlaylistIdList = this.selectedPlaylistIdList.filter(playlistId => {
+        return allPlaylistIds.includes(playlistId)
+      })
+
+      if (val > oldVal) {
+        // Focus back to search input only when playlist added
+        // Allow search and easier deselecting new created playlist
+        this.$refs.searchBar.focus()
+      }
+    },
+
+    showingCreatePlaylistPrompt(val) {
+      if (val) { return }
+
+      // Only care when CreatePlaylistPrompt hidden
+      // Shift focus from button to prevent unwanted click event
+      // due to enter key press in CreatePlaylistPrompt
+      this.$refs.searchBar.focus()
+    },
+  },
+  mounted: function () {
+    this.lastActiveElement = document.activeElement
+
+    this.updateQueryDebounce = debounce(this.updateQuery, 500)
+    // User might want to search first if they have many playlists
+    this.$refs.searchBar.focus()
+  },
+  beforeDestroy() {
+    this.lastActiveElement?.focus()
+  },
+  methods: {
+    hide: function () {
+      this.hideAddToPlaylistPrompt()
+    },
+
+    countSelected: function (playlistId) {
+      const index = this.selectedPlaylistIdList.indexOf(playlistId)
+      if (index !== -1) {
+        this.selectedPlaylistIdList.splice(index, 1)
+      } else {
+        this.selectedPlaylistIdList.push(playlistId)
+      }
+    },
+
+    addSelectedToPlaylists: function () {
+      const addedPlaylistIds = new Set()
+
+      if (this.selectedPlaylistIdList.length === 0) {
+        showToast(this.$t('User Playlists.AddVideoPrompt.Toast["You haven\'t selected any playlist yet."]'))
+        return
+      }
+
+      this.selectedPlaylistIdList.forEach((selectedPlaylistId) => {
+        const playlist = this.allPlaylists.find((list) => list._id === selectedPlaylistId)
+        if (playlist == null) { return }
+
+        this.addVideos({
+          _id: playlist._id,
+          // Use [].concat to avoid `do not mutate vuex store state outside mutation handlers`
+          videos: [].concat(this.toBeAddedToPlaylistVideoList),
+        })
+        addedPlaylistIds.add(playlist._id)
+        // Update playlist's `lastUpdatedAt`
+        this.updatePlaylist({ _id: playlist._id })
+      })
+
+      showToast(this.$t('User Playlists.AddVideoPrompt.Toast["{videoCount} video(s) added to {playlistCount} playlist(s)."]', {
```

Comment:
**suggestion (non-blocking):** use `tc` and update label with pluralization in lieu of "(s)".

Reply:
https://github.com/FreeTubeApp/FreeTube/pull/4234/commits/e7729eecc4c88a28c0385b267ac233603eeba006

There are 2 counts so 2 translation entries are used